docs/using-radicle/create-submit-patch.md

```diff
@@ -0,0 +1,67 @@
+---
+id: create-submit-patch
+title: Create and submit a patch
+---
+
+In Radicle, patches are like GitHub's pull requests. If you have a fix or enhancement you’d like to contribute to a
+project, you can do so by **submitting a patch** in Upstream. If you don't yet have Upstream, see the [get started guide](/docs/getting-started.md).
+
+First, navigate to your project's working copy with `cd`. Create and checkout a new branch that will contain the changes for your patch. The branch name should be short and descriptive – it will be used to identify the patch.
+
+```
+git checkout --branch my-proposed-changes
+```
+
+Make your changes, add the changed files with `git add ...`, and commit with `git commit ...`.
+
+Now you can create your patch using the `upstream` CLI tool, which you set up during :
+
+```
+upstream patch create
+```
+
+Your default editor opens, allowing you to enter the title and description for the patch. The first line is the title,
+followed by a blank newline, followed by the description of the patch. Reference relevant issues by including their URL
+in the patch description. Save and exit the editor.
+
+Verify that the patch shows up in Upstream by navigating to the project’s **Patches** tab in Upstream.
+
+## Inform the project's delegates about your patch
+
+First, you'll need to share your Peer ID with the delegate so that they can [track](track-review-merge.md) your patch in
+Upstream.
+
+You can find your Peer ID on the Upstream client's homepage, under your name. Click on the text to copy your Device ID
+to your system clipboard. Your Peer ID should look something like:
+`hynsqc3d693bfidom5docik9oj5hy1kciw7dz3j8pwpcnsa9cftbzk`.
+        
+![Finding your Peer ID in Upstream](/img/upstream_peer-id.png)
+    
+The delegate needs to [add your Peer ID](track-review-merge.md) to their list of remotes for the project. Once they do,
+they'll see the patch you created.
+
+At this point, the delegate will review your patch. If it passes review, it will be merged and published.
+
+## Address requested changes if needed
+
+If any changes are required, you can update and re-publish your patch. First, go to the project checkout directory, and
+make sure you're on the same branch as when you proposed the patch:
+
+```
+cd path/to/working/copy
+git checkout my-proposed-changes
+```
+
+Make the requested changes, add the changed files with `git add ...`, and commit with `git commit ...`.
+
+Now you can update the patch:
+
+```
+upstream patch update
+```
```

Comment:
This process can also be done with the CLI, but I'm not sure if we want to separate the instructions for Upstream and CLI or show them together.

Reply:
This is something I'm still working out, honestly. I'm not 100% sure whether this documentation explain how to take each action on all 3 clients—CLI, web, Upstream—or instead focus on one "optimized" workflow that uses a combination where they're necessary or where they provide the best experience. 

While maybe not something we address in this PR, I'd love to hear some ideas if you have any!

Reply:
Yep agreed it might be tough to cover with this documentation. We'll likely need some combination of various docs to cover all the workflows, and I suspect it'll change over time as things are still shifting feature-wise for every product.